src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php

```diff
@@ -105,7 +104,7 @@ private function getClassCandidates(string $class): array
                 }
             }
 
-            if ($function[0] instanceof ComposerClassLoader || $function[0] instanceof SymfonyClassLoader) {
+            if ($function[0] instanceof ComposerClassLoader || is_subclass_of($function[0], '\Symfony\Component\ClassLoader\ClassLoader')) {
```

Comment:
Should be `is_subclass_of($function[0], ClassLoader::class)`, no ?

Reply:
Or be just reverted as the previous code was just fine.

Reply:
The code refers to a non-existent class?

Reply:
Already discussed, see https://github.com/symfony/symfony/pull/25536#discussion_r157591050

Reply:
OK, let's try this: there exists a (common) situation where this code refers to a non-existent class, that's the reason PHPStan found it and complained about it.

There exists a situation where this class exists. They both exists. The new code is valid in both cases, existing code is not.

Reply:
base on what? how do you define "valid"?
on my side it's: it works as expected.

Reply:
OK. :+1: